GrandCentralBoard/Widgets/Weather/View/WeatherForecastView.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  WeatherForecastView.swift
+//  GrandCentralBoard
+//
+//  Created by Joel Fischer on 4/26/16.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import UIKit
+import GCBCore
+import CZWeatherKit
+
+struct WeatherForecastViewModel {
+    let date: String
+    let highTemperature: Int
+    let lowTemperature: Int
+    let icon: Climacon
+
+    init(model: WeatherForecastModel) {
+        // HAX: There's a bug here where the api will return the previous day as the date, at 11:58 p.m., we need to work around it
```

Comment:
Unfortunately, no. If I move it to a method and try to call it to set up the date properly in the init, it will be calling self before the parameter is initialized, which will throw an error.